index.html

```diff
@@ -1,322 +1,389 @@
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+                />
+              </svg>
+            </li>
+          </ul>
+        </nav>
+      </section>
+    </header>
+    <main class="main-container">
+      <!-- Todo: intro or headline (landing page) section  -->
+      <section class="landing flex-row">
+        <div class="headline flex-row">
+          <div class="details">
+            <h1 class="primary-head">
+              <span class="intro">Hey There.</span>
+              <span class="intro">I'm Yonas.</span>
+            </h1>
+            <div>
+              <h2 class="secondary-head">I'm a software developer</h2>
+              <p class="about-text">
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded, don’t hesitate to contact me.
+              </p>
+            </div>
           </div>
+          <ul class="share flex-row">
+            <li><img src="src/assets/icons/Vector.png" alt="Github" /></li>
+            <li><img src="src/assets/icons/Group.png" alt="Linkedin" /></li>
+            <li>
+              <img src="src/assets/icons/anglelist.png" alt="Angellist" />
+            </li>
+            <li><img src="src/assets/icons/tweeter.png" alt="Twitter" /></li>
+            <li><img src="src/assets/icons/meduin.png" alt="medium" /></li>
+          </ul>
         </div>
-        <ul class="share flex-row">
-          <li><img src="src/assets/icons/Vector.png" alt="Github" /></li>
-          <li><img src="src/assets/icons/Group.png" alt="Linkedin" /></li>
-          <li>
-            <img src="src/assets/icons/anglelist.png" alt="Angellist" />
-          </li>
-          <li><img src="src/assets/icons/tweeter.png" alt="Twitter" /></li>
-          <li><img src="src/assets/icons/meduin.png" alt="medium" /></li>
-        </ul>
-      </div>
-      <div class="illustration-container"></div>
-    </section>
-    <section class="works-section">
-      <div class="primary-work-head flex-row">
-        <h2>My Recent Works</h2>
-        <span class="line"></span>
-      </div>
-      <div class="set-grid grid">
-        <div class="container flex-row">
-          <div class="placeholder-img"></div>
-          <div class="flex-col multi-post">
-            <h3 class="third-head">Multi-Post Stories</h3>
-            <p class="daily-post-text">
-              A daily selection of privately personalized reads; no accounts
-              or sign-ups required. has been the industry's standard dummy
-              text ever since the 1500s, when an unknown printer took a
-              standard dummy text.
+        <div class="illustration-container"></div>
+      </section>
+      <!-- Todo: my projects and showcases section  -->
+      <section class="works-section">
```

Comment:
- [ Optional ] Kindly I recommend you to use `Grid`  for the work section and` flex` for align the cards in the project section.